site/public/css/electronic.css

```diff
@@ -171,13 +171,25 @@ button > i.fas:hover {
     color: initial;
 }
 
+:root {
+    --default-border-color: black;
+    --default-box-shadow: 1px 1px 2px black;
+}
+
+@media (prefers-color-scheme: dark) {
```

Comment:
Use [data-black_mode="black"][data-theme="dark"] like in colors.css

Reply:
Michael cannot see the css if it does this, I do not know why.

Reply:
Update you checked the OS's color scheme instead of Submitty light scheme, comment still stands.

Reply:
got it! just pushed up the changes. Please check